### PR TITLE
Removes public sandbox information

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-1306](https://openedx.atlassian.net/browse/DOC-1306)

The point of doing this PR was to clarify that you CAN access Studio in the public sandbox, but you can't do it directly from the LMS. Then, I started doing some cleanup. 
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: @feanil 
- [ ] Subject matter expert: @andy-armstrong 
- [ ] Doc team review (sanity check): @pdesjardins @srpearce note that I did not do a complete overhaul, more like a whitewashing 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

File: en_us/developers/source/public_sandboxes.rst [hunk 1]
```diff
@@ -1,46 +1,69 @@
...
+course team role, use the following credentials.
 
 * username: staff@example.com
 * password: edx
 
-You can log in to a student account using one the following credentials.
-These user accounts represent students enrolled in the demo course in the
-audit, honor code, or verified certificate tracks, respectively.
+This user account has the Admin course team role for the edX Demonstration
+course. You can use to view the complete course, including the instructor
```
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: Typo: "You can use to view"

File: en_us/developers/source/public_sandboxes.rst [hunk 2]
```diff
@@ -58,104 +81,114 @@ types of translation errors. Use
...
-because the CSS needs to be compiled separately (fixing this issue is a task on our
-backlog!).
+because the CSS needs to be compiled separately (fixing this issue is a task on
+our backlog!).
 
-Note: This is our first deployment of our alpha version of RTL language support! If
-you have any comments or find any visual bugs, please let us know by posting on the
-`openedx-translation <https://groups.google.com/forum/#!forum/openedx-translation>`_
-mailing list.
+.. note:: This is our first deployment of our alpha version of RTL language
```
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: Is our RTL language support still only an alpha version?

File: en_us/developers/source/public_sandboxes.rst [hunk 3]
```diff
@@ -58,104 +81,114 @@ types of translation errors. Use
...
+  support! If you have any comments or find any visual bugs, please let us know
+  by posting on the `openedx-translation
+  <https://groups.google.com/forum/#!forum/openedx-translation>`_ mailing list.
 
-LTR and RTL sandboxes are available for both the LMS, or learning managment system (the part
-of the website that students see) and Studio, the course authoring platform.
-You can access the LMS at:
+LTR and RTL sandboxes are available for both the LMS (learning management
+system), which is the part of the website that students see, and Studio, the course authoring platform. You can access the LMS at these URLs.
```
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: Nit: split the long line?

File: en_us/developers/source/public_sandboxes.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: how could I find out the answer to that, @andy-armstrong ?

File: en_us/developers/source/public_sandboxes.rst [hunk 4]
```diff
@@ -58,104 +81,114 @@ types of translation errors. Use
...
+you are encouraged to make new courses, as well as add and delete course
+content. The sandboxes can be reset if anything breaks, and they are completely
+disconnected from the production version of the edx.org website.
 
+====================
 Caveats and Warnings
 ====================
-#. These sandboxes will be updated with new translations and the newest version
+
+#. These sandboxes are updated with new translations and the newest version
```
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: Are we still updating the sandboxes once a week?

File: en_us/developers/source/public_sandboxes.rst [hunk 5]
```diff
@@ -58,104 +81,114 @@ types of translation errors. Use
...
+   instead of ``<b> </b>``).
 
-We hope you find these sandboxes helpful. If you have any questions, comments, or
-concerns, please give us feedback by posting on the
-`openedx-translation <https://groups.google.com/forum/#!forum/openedx-translation>`_
-mailing list. We'd be happy to hear about any improvements you think we could make!
+We hope you find these sandboxes helpful. If you have any questions, comments,
+or concerns, please give us feedback by posting on the `openedx-translation
+<https://groups.google.com/forum/#!forum/openedx-translation>`_ mailing list.
+We are happy to hear about any improvements you think we could make!
```
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: You could also mention the new #i18n slack channel.

File: en_us/developers/source/public_sandboxes.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: @andy-armstrong how can I find out the answer?

File: en_us/developers/source/public_sandboxes.rst (hunk 5 above)
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: good point. I may send them to the list of contact points on the portal

File: en_us/developers/source/public_sandboxes.rst (hunk 2 above)
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: I don't know who owns right-to-left support. @jbarciauskas is this something your team owns?

File: en_us/developers/source/public_sandboxes.rst (hunk 4 above)
Comment (andy-armstrong) [inline en_us/developers/source/public_sandboxes.rst]: @feanil are the sandboxes updated on a weekly schedule, or is it more ad hoc?

File: en_us/developers/source/public_sandboxes.rst (hunk 2 above)
Comment (jbarciauskas) [inline en_us/developers/source/public_sandboxes.rst]: @andy-armstrong I think solutions or T&L owns it for the platform. I know there's been a project related on the Solutions side but not sure where it is.

File: en_us/developers/source/public_sandboxes.rst [hunk 6]
```diff
@@ -58,104 +81,114 @@ types of translation errors. Use
...
+our backlog!).
 
-Note: This is our first deployment of our alpha version of RTL language support! If
-you have any comments or find any visual bugs, please let us know by posting on the
-`openedx-translation <https://groups.google.com/forum/#!forum/openedx-translation>`_
-mailing list.
+.. note:: This is our first deployment of our alpha version of RTL language
+  support! If you have any comments or find any visual bugs, please let us know
+  by posting on the `openedx-translation
+  <https://groups.google.com/forum/#!forum/openedx-translation>`_ mailing list.
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: @nedbat can I delete the first sentence in this note? are we past alpha/initial deployment?

File: en_us/developers/source/public_sandboxes.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: @andy-armstrong @jbarciauskas any objections if I open a new doc story to track down the alpha/non-alpha status of RTL support? or is that a blocker for this PR?

File: en_us/developers/source/public_sandboxes.rst [hunk 7]
```diff
@@ -1,46 +1,69 @@
...
 
-The sandbox provides course team and student-level access to a copy of the
-current version of the edx.org website. This sandbox does not allow access to
-Studio, the course authoring system.
+The edx.org sandbox is intended for those looking to try out the software that
+powers `edx.org <www.edx.org>`_. This sandbox provides course team and student-
+level access to the software that powers the edx.org website.
 
-You can log in by visiting the following URL.
+You can sign in by entering the `https://www.sandbox.edx.org/
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: To sign in to the LMS, enter...

File: en_us/developers/source/public_sandboxes.rst [hunk 8]
```diff
@@ -1,46 +1,69 @@
...
 * username: staff@example.com
 * password: edx
 
-You can log in to a student account using one the following credentials.
-These user accounts represent students enrolled in the demo course in the
-audit, honor code, or verified certificate tracks, respectively.
+This user account has the Admin course team role for the edX Demonstration
+course. You can use to view the complete course, including the instructor
+dashboard, in the LMS. In Studio, you can add to and change the content of the
+Demo course.
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: give the studio URL

File: en_us/developers/source/public_sandboxes.rst [hunk 9]
```diff
@@ -1,46 +1,69 @@
...
+
+.. note:: The edx.org sandbox does not offer the View in Studio feature that
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: bold View in Studio

File: en_us/developers/source/public_sandboxes.rst [hunk 10]
```diff
@@ -1,46 +1,69 @@
...
+ allows you to navigate from course content in the LMS directly to the
+ corresponding component in Studio. You must sign in to Studio and the LMS
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: Instead, you can sign in ...

File: en_us/developers/source/public_sandboxes.rst [hunk 11]
```diff
@@ -1,46 +1,69 @@
...
+powers `edx.org <www.edx.org>`_. This sandbox provides course team and student-
+level access to the software that powers the edx.org website.
 
-You can log in by visiting the following URL.
+You can sign in by entering the `https://www.sandbox.edx.org/
+<https://www.sandbox.edx.org/>`_ URL in a web browser.
 
-* `https://www.sandbox.edx.org/ <https://www.sandbox.edx.org/>`_
+==================================
+Signing in as a Course Team Member
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: u/c In

File: en_us/developers/source/public_sandboxes.rst [hunk 12]
```diff
@@ -1,46 +1,69 @@
...
+dashboard, in the LMS. In Studio, you can add to and change the content of the
+Demo course.
+
+.. note:: The edx.org sandbox does not offer the View in Studio feature that
+ allows you to navigate from course content in the LMS directly to the
+ corresponding component in Studio. You must sign in to Studio and the LMS
+ separately.
+
+==================================
+Signing in as a Learner
```
Comment (lamagnifica) [inline en_us/developers/source/public_sandboxes.rst]: u/c In

**Comment by lamagnifica** [comment]
Per Fred, DevOps no longer maintains the public sandbox. Per Ned, the language sandboxes have not been in use for over 7 months. Deleting this section instead of revising it. Thumbs from Steve Magoun as well.

**Comment by lamagnifica** [comment]
@srpearce @pdesjardins or @catong would you please sanity check this? (I couldn't resist cleaning up that translator's file, but any further work on it is way outside the scope of the original tiny fix I started out trying to do...)

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 13]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
 
-You should become a translator only if you feel very confident
-in both your language in English. Open edX is written in English, so all
-translations will be done from English to your language.
+.. note:: All strings must be both translated **and reviewed** before
+  we will publish them on the edx.org website.
+
+You should become a translator only if you feel very confident in both your own
+language and in English. Open edX is written in English, so all translations
+are done from English to your language.
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: done from English to your language > done from English to your other language

Sounds alienating as is.

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 14]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
-* `Joining A Translation Team`_
-* `Joining A Review Team`_
-* `How To Translate With Transifex`_
-* `Requesting A New Language`_
+.. contents::
+ :local:
+ :depth: 1
 
-Signing Up For An Account
+Sign Up for an Account
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: Non-sanity, ignore: I think maybe "up" should be lower case.

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 15]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
+Optionally, you can request to become a `reviewer
+<http://support.transifex.com/customer/portal/articles/1167280>`_ for a
+language. Review teams are incredibly important, because all strings must be
+both translated **and reviewed** before we will publish them on the edx.org
+website.
 
-You should join a review team only if you feel very confident in both your language
-and in English. To join a review team, request permission from the project coordinator(s).
+You should join a review team only if you feel very confident in both your
+language and in English. To join a review team, request permission from the
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst:94]: "your language" opposed to "English" reads badly to me. Not globally welcoming.

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 16]
```diff
@@ -124,109 +134,115 @@ translate and review these resources, in this order, before anything else:
...
-and ``>`` characters.
+You should translate the text that is between the tags. HTML markup tags begin
+and end with ``<`` and ``>`` characters.
 
-Spacing is especially important. Adding spaces in an HTML tag (eg changing ``</a>`` to ``</ a>``)
-will break the website.
+Spacing is especially important. Adding spaces in an HTML tag (for example,
+changing ``</a>`` to ``</ a>``) will break the website.
 
 Examples::
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst:217]: Is that supposed to be a Sphinx directive? Or should there be one colon? Looks like that might render as a pseudo-heading with two colons.

Stet, it's a Sphinx directive, although I couldn't Google it up easily. Looks OK in HTML.

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 17]
```diff
@@ -333,11 +369,12 @@ Please continue reading for examples of each type of placeholder form inside a s
...
-3. Do **not** alter or translate placeholder strings that appear within a ``<%= %>`` block. Placeholder
-   strings in this format look like this: ``<%= student_name %>``.
+#. Do not alter or translate placeholder strings that appear within a
+   ``<%= %>`` block. Placeholder strings in this format look like this:
+   ``<%= student_name %>``.
 
-   As in the previous examples, you must not add, change, or remove punctuation, change capitalization,
-   or add in new spacing.
+   As in the previous examples, you must not add, change, or remove
+   punctuation, change capitalization, or add in new spacing.
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst:390]: Non-sanity: add in > add

File: en_us/shared/preface.rst [hunk 18]
```diff
@@ -283,12 +283,12 @@ Documentation
 
 Documentation for developers is available on the `docs.edx.org`_ web page.
 
-* The `edX Platform Developer's Guide`_ includes guidelines for
-  contributing to Open edX, options for extending the Open edX platform, using
-  the edX public sandboxes, instrumenting analytics, and testing.
+* The `edX Platform Developer's Guide`_ includes guidelines for contributing to
+  Open edX, options for extending the Open edX platform, instrumenting
```
Comment (pdesjardins) [inline en_us/shared/preface.rst]: contributing to the Open edX project, options ...

Per recent trademark usage discussion?

File: en_us/shared/preface.rst [hunk 19]
```diff
@@ -382,9 +382,9 @@ is available.
 * `Installing, Configuring, and Running the Open edX Platform`_ provides
   information about installing and using devstack and fullstack.
 
-* The `edX Platform Developer's Guide`_ includes guidelines for
-  contributing to Open edX, options for extending the Open edX platform, using
-  the edX public sandboxes, instrumenting analytics, and testing.
+* The `edX Platform Developer's Guide`_ includes guidelines for contributing to
+  Open edX, options for extending the Open edX platform, instrumenting
```
Comment (pdesjardins) [inline en_us/shared/preface.rst]: Again: contributing to Open edX > contributing to the Open edX project?

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 20]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
+   the translation team for that language.
 
    .. image:: /images/join-language-team.png
+     :alt: The Japanese language page on the Transifex website, with the Join
+         team option indicated.
 
-   Once you've joined a translation team, you can begin translating strings immediately! See the
-   section `How To Translate With Transifex`_.
+   After you join a translation team, you can begin translating strings
+   immediately. For more information, see `How To Translate With Transifex`_.
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: The tests fail on this link.

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 21]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
+The project maintainers appear at the top of the main project page.
 
-.. image:: /images/project-maintainers.png
+If you are willing to become a coordinator for your language, let us know in
+your message.
 
-If you are willing to become a coordinator for your language, let us know in your message.
-
-How To Translate With Transifex
+Translating With Transifex
```
Comment (pdesjardins) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst:120]: Looks like changing this title broke that horribly brittle cross-reference to it. Drop in an anchor with the target string "How To Translate With Transifex"?

**Comment by pdesjardins** [comment]
I added some minor comments, there's a broken link problem to fix, otherwise sane! 👍

File: en_us/shared/preface.rst (hunk 18 above)
Comment (jbarciauskas) [inline en_us/shared/preface.rst]: Yes to both of these, good catch @pdesjardins !

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 22]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
-`public translation sandboxes <public_sandboxes.rst#language-sandboxes>`_.
+.. contents::
+ :local:
+ :depth: 1
 
+******************************
 Getting Started with Transifex
 ******************************
 
 Contributors wishing to help translate edx-platform and
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: "Contributors wishing to" -> "Contributors who want to..."

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst (hunk 13 above)
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: Or perhaps "from English to your target language"?

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 23]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
-translations will be done from English to your language.
+.. note:: All strings must be both translated **and reviewed** before
+  we will publish them on the edx.org website.
+
+You should become a translator only if you feel very confident in both your own
+language and in English. Open edX is written in English, so all translations
+are done from English to your language.
 
 This guide shows images from the Transifex website in English. Depending
 on your browser settings, you may see the site in a different language. The
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: "may" -> "might"

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 24]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
 button locations will be the same.
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: "button locations will be the same" -> "button locations are the same"

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 25]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
 
-1. Once you've set up your account, visit the `edx-platform project <https://www.transifex.com/projects/p/edx-platform/>`_
+#. After you set up your account, visit the `edx-platform project
+   <https://www.transifex.com/projects/p/edx-platform/>`_
    to become a translator for your language.
 
-2. When you visit the project page, it looks like this.  If your language is listed, click
-   on the name of the language to go to the language's page. Here is an example of choosing
-   the Japanese translation project:
+#. The project page lists different languages. If your language is
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: Doesn't really explain what is being listed. Is it a list of the languages with existing translation projects?

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst (hunk 15 above)
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst:94]: Perhaps "target language" as suggested above as well.

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 26]
```diff
@@ -13,108 +13,118 @@ for translators to follow when executing translations. For further discussion,
...
+website.
 
-You should join a review team only if you feel very confident in both your language
-and in English. To join a review team, request permission from the project coordinator(s).
+You should join a review team only if you feel very confident in both your
+language and in English. To join a review team, request permission from the
+project coordinator(s).
 
-1. Click on the "Members" tab of your language:
+#. On your Transifex edx-platform project page for your language, select
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: Same suggestion re "target language" vs "your language"

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 27]
```diff
@@ -124,109 +134,115 @@ translate and review these resources, in this order, before anything else:
...
+If your language is not listed on the Transifex edx-platform project page,
+select **Request language** to start a new translation project for your
+language.
 
-Before requesting a new language, please first make sure your language is not listed! Keep in
-mind that variants of your language may exist ("Chinese" versus "Chinese-China") and these
-variants may meet your needs.
+Before you request a new language, please make sure your language is not
+listed. Keep in mind that variants of your language might exist ("Chinese"
+versus "Chinese-China") and one of these variants may meet your needs.
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: "may meet" - > "might meet"

File: en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst [hunk 28]
```diff
@@ -124,109 +134,115 @@ translate and review these resources, in this order, before anything else:
...
-An edX translation team member will respond to your request within a few days. If your
-language is approved, you will be the coordinator of the project. You can add additional
-coordinators, reviewers, and translators as you wish.
-
-If you request a new language, we ask that you commit to the success of your language's
-translation project. Particularly, we expect you to be an active translator and work to
-recruit other translators and reviewers so your language project advances to completion,
-with all strings translated and reviewed, so that we can publish your work on the edx.org website.
+An edX translation team member will respond to your request within a few days.
+If your language is approved, you will be the coordinator of the project. You
```
Comment (catong) [inline en_us/developers/source/conventions/internationalization/i18n_translators_guide.rst]: "you will be the coordinator" -> "you become the coordinator"

**Comment by lamagnifica** [comment]
new commit @pdesjardins @catong

**Comment by pdesjardins** [comment]
Even better, 👍 .

**Comment by catong** [comment]
👍 also!

**Pull request merged**